shared/fs/sortbar.js

```diff
@@ -1,12 +1,47 @@
 // @flow
 import * as React from 'react'
 import {globalStyles, globalColors, isMobile} from '../styles'
-import {Box, Divider} from '../common-adapters'
+import {Box, ClickableBox, Divider} from '../common-adapters'
+import * as Constants from '../constants/fs'
+import * as Types from '../constants/types/fs'
 
-const SortBar = () => (
+export type OnSortSettingChange = (setting: Types._SortSetting) => void
+
+export type SortBarProps = {
+  onSortSettingChange: OnSortSettingChange,
+  sortSetting: Types._SortSetting,
+}
+
+export const SortBar = ({onSortSettingChange, sortSetting}: SortBarProps) => (
   <Box>
     <Divider />
-    <Box style={stylesSortBar} />
+    <Box style={stylesSortBar}>
+      Super Ugly Temporary SortBar:
+      <ClickableBox
+        onClick={() =>
+          onSortSettingChange(
+            Constants.makeSortSetting({
+              sortBy: sortSetting.sortBy,
+              sortOrder: sortSetting.sortOrder === 'asc' ? 'desc' : 'asc',
+            })
+          )
+        }
+        style={{marginLeft: '8px', color: 'red'}}
+      >
+        Click to change sort-order: {sortSetting.sortOrder}
```

Comment:
Same as above, if you put this into a `<Text type="BodySmall">`, it won't be ugly any more.

Reply:
Magical!